packages/react-kit/src/components/Popover/Popover.tsx

```diff
@@ -569,35 +613,62 @@ function movePopoverVertically(
 	}
 
 	return undefined;
-}
+};
+
+type THorizontalMoveProps = {
+	placement: PopoverPlacement;
+	align: PopoverAlign;
+	anchorLeft: number;
+	anchorRight: number;
+	popoverWidth: number;
+	parentWidth: number;
+	checkBounds?: Boolean;
+	previousPlacement?: PopoverPlacement;
+};
 
 type THorizontalPosition = {
 	left: number;
 	placement: PopoverPlacement;
 	align: PopoverAlign;
 };
 
-function movePopoverHorizontally(
-	placement: PopoverPlacement,
-	align: PopoverAlign,
-	anchorLeft: number,
-	anchorRight: number,
-	popoverWidth: number,
-	parentWidth: number,
+const movePopoverHorizontally: (props: THorizontalMoveProps) => THorizontalPosition | undefined = ({
```

Comment:
for what purpose you replaced usual function with arrow one?

Reply:
I returned the simple function.